save-backend/src/main/kotlin/org/cqfn/save/backend/controllers/UsersDetailsController.kt

```diff
@@ -0,0 +1,49 @@
+package org.cqfn.save.backend.controllers
```

Comment:
Since `username` can be similar for different sources (github, gittee), this logic should be probably reworked

Reply:
Yes. I think we will fix this later, when the Source field will be filled in for users